framework/src/modules/token/module.ts

```diff
@@ -59,7 +67,11 @@ export class TokenModule extends BaseModule {
 		const genesisStore = codec.decode<GenesisTokenStore>(genesisTokenStoreSchema, assetBytes);
 		const userStore = context.getStore(this.id, STORE_PREFIX_USER);
 		for (const userData of genesisStore.userSubstore) {
-			await userStore.setWithSchema(userData.address, userData, userStoreSchema);
+			await userStore.setWithSchema(
+				getUserStoreKey(userData.address, userData.tokenID),
```

Comment:
Temporally update the genesis block handling. Proper implementation will be done in #7119